client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -57,7 +57,7 @@ const ManagePluginsItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 		: [ getPluginsUrl( site.slug ), __( 'Plugins' ) ];
 
 	return (
-		<PopoverMenuItem
+		<MenuItem
 			href={ href }
```

Comment:
This `href` is throwing a TypeScript warning and I don't know why 😦

Reply:
- @danielbachhuber, here is a way to "fix" it #66877

Ideally, we should use another component that supports `href` or we should change the types on https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wordpress__components/button/index.d.ts#L134

Reply:
Fixed with #66879

Reply:
... and thanks for your help, @sejas !

Reply:
Yes there's definitely just a type error here. Tracing through the Gutenberg code I found `<MenuItem>` is just a wrapper around `<Button>` which explicitly accepts an `href` prop.